frontend/asset_mapper.rst

```diff
@@ -207,6 +212,8 @@ This adds the ``bootstrap`` package to your ``importmap.php`` file::
     such as ``@popperjs/core``. The ``importmap:require`` command will add both the
     main package *and* its dependencies. If a package includes a main CSS file,
     that will also be added (see :ref:`Handling 3rd-Party CSS <asset-mapper-3rd-party-css>`).
+    If the associated CSS file isn't added to the importmap, you can contact the package
```

Comment:
This should be in the "handling 3rd party" section, as this paragraph states "if a packages includes a CSS files".